README.md

```diff
@@ -139,6 +139,20 @@ if (!$manager->userHasPermission($anotherUserId, 'viewList', ['action' => 'home'
 }
 ```
 
+## Advanced usage:
+
+#### Create an instance and using other storage for user assignments
+
+if you want store user assignments in db or other storage
+
+For detail information - see [Issue 49](https://github.com/yiisoft/rbac/issues/49)
+
+```php
+$manager = new Manager($storage, new ClassNameRuleFactory());
+$manager->setAssignmentStorage($otherStorage);
```

Comment:
Please include $storage and $otherStorage instantiation code, and name the variables accordingly, to ease the comprehension for beginners ($phpStorage = .., $dbStorage = ..)

Reply:
Sorry, nut now we fon't have released implementation of rbac-db. After release - documentation will be updated.